packages/cfd/src/Containers/cfd-reset-password-modal.tsx

```diff
@@ -13,7 +13,7 @@ const ResetPasswordIntent = ({ current_list, children, is_eu, ...props }: TReset
     const reset_password_type = localStorage.getItem('cfd_reset_password_type') || 'main'; // Default to main
     const has_intent =
         reset_password_intent &&
-        /(real|demo)\.(financial_stp|financial|synthetic|synthetic_svg|synthetic_bvi|financial_svg|financial_bvi|financial_fx|financial_v)/.test(
+        /(real|demo)\.(financial_stp|financial|synthetic|synthetic_svg|synthetic_bvi|financial_svg|financial_bvi|financial_fx|financial_v|all_svg|dxtrade)/.test(
```

Comment:
@shontzu-deriv how about `all-demo`?

Reply:
for Demo Account i think it is just `demo` instead of `all-demo`